README.md

```diff
@@ -26,57 +26,26 @@ Differentiating features include:
 [YAML]: http://yaml.org
 [Prometheus]: http://prometheus.io
 
-## Architecture
+## Why Chihaya?
 
-### Diagram
+Chihaya is built for developers who need to deploy a BitTorrent tracker, but have additional requirements than general purpose public trackers like those operated by [OpenBitTorrent].
```

Comment:
I'm not a native speaker, the construct "additional requirements than something" sounds strange to me, is it correct?